Core/include/Acts/Surfaces/CylinderSurface.hpp

```diff
@@ -38,7 +38,7 @@ class DetectorElementBase;
 /// since it builds the surfaces of all TrackingVolumes at container level
 /// for a cylindrical tracking geometry.
 ///
-/// @image html figures/CylinderSurface.png
+/// @image html /figures/CylinderSurface.png
```

Comment:
I still don't see the image unfortunately... 
![image](https://github.com/acts-project/acts/assets/72298366/4b1b0b0e-6017-4a97-bb78-0ec832c298ed)

Reply:
@paulgessinger do you know what the problem could be? I would have thought that doxygen would crash if it does not find an image link

Reply:
It's rewriting them a bunch of times when copying them around, so not sure.

Reply:
looks like this was already broken https://acts.readthedocs.io/en/latest/api/class/class_acts_1_1_cylinder_surface.html?highlight=cylindersurface